corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -661,15 +738,16 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
                 // Update models based on response
                 if (response.queryResponse) {
                     _.each(response.queryResponse.displays, function (responseModel, i) {
-                        self.collection.models[i].set({
+                        self._getChildModels().set({
```

Comment:
Is this `.set` is working on a group of models now instead of an individual one? (and does that also mean `i` from the line above is unused?)

Reply:
That was supposed to be `self._getChildModels()[i].set({`